docs/intro-to-lora.md

```diff
@@ -1,81 +1,79 @@
+For hardware, you need devices and gateways, similar to how you would set up a Wi-Fi network. Gateways scan the spectrum and capture LoRa packets. There is no gateway pinning here - devices are not associated with a single gateway; thus, all gateways within range of a device receive the signal. The gateways then forward their data to a network service that handles the packet.
 
-The network service de-duplicates packets when multiple gateways receive the same packet, decrypts the message (everything is end-to-end encrypted), handles LoRa features like adaptive data rating, and so on. It then forwards the decrypted data to your application.
+The network service deduplicates packets when multiple gateways receive the same packet, decrypts the message (everything is end-to-end encrypted), handles LoRa features such as adaptive data rating and so on. It then forwards the decrypted data to your application.
 
-That gives us five requirements.
+There are five requirements.
 
-We need hardware:
+Hardware:
 
-* Gateways
-* Devices
+* Gateways.
+* Devices.
 
-And we need software:
+Software:
 
-* Device firmware
-* A network service
-* An app
+* Device firmware.
+* A network service.
+* An app.
 
-In this guide we'll show you which hardware you can buy, and we'll use two online services that will make it easy to write device firmware and handle your LoRa traffic.
+This guide shows you which hardware you can buy and two online services you can use to write device firmware and handle your LoRa traffic.
 
 ### Getting a gateway
 
-There's quite some [choice in the gateways](https://www.loriot.io/gateways.html) we can use, but I've had good experience with these three:
+You have [choices in the gateways](https://www.loriot.io/gateways.html) you can use, but we've had good experience with these three:
 
-* [Kerlink IoT station](http://www.kerlink.fr/en/products/lora-iot-station-2/lora-iot-station-868-mhz). Expensive (around 1,200 euros), but great build quality and range.
-* [Multitech Conduit](http://www.multitech.com/brands/multiconnect-conduit). About a third of the price of the Kerlink (around 450 euros), and pretty great for small setups (put a bigger antenna on it though). Multitech also has a [rugged outdoor](http://www.multitech.com/brands/multiconnect-conduit-ip67) version.
-* Building your own with a Raspberry Pi and an [IMST iC880A](http://webshop.imst.de/catalogsearch/result/?q=iC880A) concentrator. At around 230 euros, this is the most cost-efficient option.
+* [Kerlink IoT station](http://www.kerlink.fr/en/products/lora-iot-station-2/lora-iot-station-868-mhz). Expensive (around 1,200 euros) but great build quality and range.
```

Comment:
I can't get the Kerlink IoT station link to work. Could you please confirm it is correct?